examples/purchase-tester/maestro-tests/play_store_login.yaml

```diff
@@ -0,0 +1,20 @@
+appId: com.revenuecat.purchases_sample
+---
+- launchApp:
+      appId: "com.android.vending"
+      clearState: true
+- tapOn:
+      text: "Sign in"
+- tapOn:
+      # This is the id for the inputText in the play store.
+      id: "identifierId"
+- inputText: ${GOOGLE_EMAIL}
+- hideKeyboard
```

Comment:
The `hideKeyboard` command seems to also input a "Enter" key command. I tried to tap on the "Next" button but that was doing weird things with Maestro, like tapping in incorrect places... This seemed to work better.